src/notification/notification-drawer.html

```diff
@@ -1,7 +1,7 @@
 <div class="drawer-pf" ng-class="{'hide': $ctrl.drawerHidden, 'drawer-pf-expanded': $ctrl.drawerExpanded}">
   <div  ng-if="$ctrl.drawerTitle" class="drawer-pf-title">
-    <a ng-if="$ctrl.allowExpand" class="drawer-pf-toggle-expand fa fa-angle-double-left" ng-click="$ctrl.toggleExpandDrawer()"></a>
-    <a  ng-if="$ctrl.onClose" class="drawer-pf-close pficon pficon-close" ng-click="$ctrl.onClose()"></a>
+    <a href="javascript:void(0);" ng-if="$ctrl.allowExpand" class="drawer-pf-toggle-expand fa fa-angle-double-left" ng-click="$ctrl.toggleExpandDrawer()"></a>
+    <a href="javascript:void(0);" ng-if="$ctrl.onClose" class="drawer-pf-close pficon pficon-close" ng-click="$ctrl.onClose()"></a>
```

Comment:
We need to figure out the unit tests and use href="" or, worst case, use href="#0"

Reply:
getting the same broken test with href="#0" as well